### PR TITLE
Keystone update

### PR DESCRIPTION
**Pull request by jdextercsc**
Updates to keystone role to allow use UUID and multi-threading capability via httpd

### CODE REVIEW

**Comment by jdextercsc** [comment]
Update to keystone to use UUID and the httpd core for multi threading

**Comment by jdextercsc** [comment]
```
Update to keystone to use UUID and the httpd core for multi threading
```

File: roles/keystone/tasks/keystone_juno.yml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  with_items:
+    - openstack-keystone
+    - openstack-utils
+    - httpd
+  tags: keystone
+
+- name: set keystone configuration
+  command: openstack-config {{ item }}
+  with_items:
+    - " --set /etc/keystone/keystone.conf DEFAULT admin_token {{keystone_admin_token}}"
```
Comment (jcpowermac) [inline roles/keystone/tasks/keystone_juno.yml:15]: This won't be idempotent if `--verbose` and `changed_when` isn't used.  Do we care at this point?  Or v2?

Comment (jdextercsc) [inline roles/keystone/tasks/keystone_juno.yml:15]: The openstack-config handles a lot of that. It  looks at the ini file and if it needs to change the line it does. If it doesnt exist it adds it.

**Comment by jcpowermac** [comment]
Looks good to me :+1:

**Comment by mtnbikenc** [comment]
Looks good :+1:

**Pull request merged**